apps/frontend/src/pages/plan-style-overview/lib/helpers.ts

```diff
@@ -0,0 +1,124 @@
+import H2C from "html2canvas";
```

Comment:
rename the file. It should not be a general helper but a specific downloadPlan service